src/dependencies/useStoreDependency.ts

```diff
@@ -67,15 +76,13 @@ function useStoreDependency<Props>(
     dispatchToken,
```

Comment:
Since `dispatchToken`, `dependencyValue` and `dependencyMap` are refs I actually think you can remove them from the deps array. I'm struggling to find the docs to make note of this.

Reply:
yeah, you're right, no complaints from the lint rule about getting rid of refs in the deps array